SPECS/kubernetes/kubernetes-1.18.14.spec

```diff
@@ -10,7 +10,7 @@
 Summary:        Microsoft Kubernetes
 Name:           kubernetes
 Version:        1.18.14
-Release:        5%{?dist}
+Release:        6%{?dist}
```

Comment:
Heads-up: all `kubernetes` specs are gone from `dev` now, so you can safely take the option to delete them when you're going to merge in the latest `dev.